dspace-uclouvain/src/main/java/org/dspace/uclouvain/core/ConfigurationFileLoader.java

```diff
@@ -0,0 +1,81 @@
+package org.dspace.uclouvain.core;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * Class that can be used to deal with the loading of a file's data.
+ * This class allows to check for new version of the file through the ´isFileNewer´ method.
+ * You can also read file's data through the ´readFileAsString´ && ´readFileAsBytes´ methods.
+ */
+public class ConfigurationFileLoader {
+    // Time in ms of the last modification of the file
+    private Long lastModified;
+    private String absoluteFilePath;
+    public boolean isError = false;
+
+    public ConfigurationFileLoader(String filePath){
+        this.absoluteFilePath = filePath;
+        this.loadFile();
+    }
+
+    /**
+     * Read the file and set the 'this.isError' && 'this.lastModified' properties.
+     * If the file does not exists or cannot be red, set error to true. 
+     * If the file exists and can be red, set the lastModified ms to the file's last modified date.
+     */
+    private void loadFile(){
+        File configurationFile = new File(this.absoluteFilePath);
+        if (configurationFile.exists() && configurationFile.canRead()){
+            this.lastModified = configurationFile.lastModified();
+            this.isError = false;
+        } else {
+            this.isError = true;
+        }
+    }
```

Comment:
Don't think it's a smart idea to keep `isError` flag ; just throws a `IOException` is files doesn't exists or can't be read.